README.md

```diff
@@ -129,6 +131,23 @@ store.dispatch(fetch({
 }))
 ```
 
+#### abort fetch
+
+Abort a fetch by dispatching `abortFetch({type, id})`.
+
+```js
+const fetch, {abortFetch, getID} = require('@conveyal/woonerf/fetch')
+const id = getID()
+store.dispatch(fetch({
+  url: 'http://conveyal.com',
+  id,
+  next: () => {
+    throw new Error('Will not be called')
+  }
+}))
+store.dispatch(fetch(abortFetch({id})))
```

Comment:
I think the `fetch` call in here shouldn't be here.

Reply:
True